source/draft_users.rst

```diff
@@ -13,85 +13,130 @@ Users are managed primarily through the API, by admin users.
 Users Model
 -----------
 
-For reference of the structure of this table, see the :ref:`draft_model<draft_model>` docs.
-
-============  ========  =============  ==============================  =============================================
-    Name        Type    POST Required           Description                                Notes
-============  ========  =============  ==============================  =============================================
-username      string        true       Username the user logs in with  Used for display if displayname is null
-password      string        true       Password for user login         Stored as bcrypt hash; not returned on get
-display_name  string        false      Name used for public display
-active        boolean       false      Whether user is can log in      Used by admins to invalidate users
-admin         boolean       false      Whether user has admin rights   Can only be set by another admin
-email         string        false      Email address of user
-created_at    ISO date      false      The date the user was created   Automatically set by the server, unchangeable
-deleted_at    ISO date      false      The date the user was deleted   Automatically set by the server upon delete
-============  ========  =============  ==============================  =============================================
-
-It is worth noting that, because users are updated "in-place" (i.e. without an audit trail
-or revision system), there are no ``uuid``, ``revision``, or ``updated_at`` fields.
+For reference of the structure of this table, see the
+:ref:`draft_model<draft_model>` docs.
+
+Here is an example user object:
 
 -----------
 
 Admin Users
 -----------
 
-An admin user is defined by the boolean ``admin`` field. Admin users are able to access
-any endpoint, and manage all users (including being the only users who can add users or
-promote them to admin).
+A site-wide admin user is defined by the boolean ``admin`` field. Admins
+are able to access any endpoint, and manage all users (including being the only
+users who can add users or promote them to admin).
 
 ---------------
 
 /users Endpoint
 ---------------
 
-User management involves a new endpoint, at ``/users``. In most respects it is similar to
-the other endpoints, with certain key differences.
+User management involves a new endpoint, at ``/users``. In most respects it is
+similar to the other endpoints, with certain key differences.
 
 GET /users
 ~~~~~~~~~~
 
-Returns a list of all users in the database:
+Returns a list of all user objects, sorted alphabetically by username.
 
 .. code-block:: javascript
 
     [
       {
-        "username": "example",
-        "displayname": "X. Ample User",
-        "email": "example@example.com",
+        "display_name": "User One",
+        "username": "user1",
+        "email": "user1@example.org",
+        "site_wide_permissions": {
```

Comment:
Did we decide we wanted an object to hold permissions? I thought they were just going to be flat fields on the user.